packages/plugin-ext/src/common/known-commands.ts

```diff
@@ -0,0 +1,315 @@
+    mappings['editor.action.jumpToBracket'] = ['monaco.editor.action.jumpToBracket', createConversionFunction(...[])];
+    mappings['actions.findWithSelection'] = ['monaco.actions.findWithSelection', createConversionFunction(...[])];
+    mappings['editor.action.nextMatchFindAction'] = ['monaco.editor.action.nextMatchFindAction', createConversionFunction(...[])];
+    mappings['editor.action.previousMatchFindAction'] = ['monaco.editor.action.previousMatchFindAction', createConversionFunction(...[])];
+    mappings['editor.action.nextSelectionMatchFindAction'] = ['monaco.editor.action.nextSelectionMatchFindAction', createConversionFunction(...[])];
+    mappings['editor.action.previousSelectionMatchFindAction'] = ['monaco.editor.action.previousSelectionMatchFindAction', createConversionFunction(...[])];
+    mappings['editor.action.openLink'] = ['monaco.editor.action.openLink', createConversionFunction(...[])];
+    mappings['editor.action.quickFix'] = ['monaco.editor.action.quickFix', createConversionFunction(...[])];
+    mappings['editor.action.refactor'] = ['monaco.editor.action.refactor', createConversionFunction(...[])];
+    mappings['editor.action.sourceAction'] = ['monaco.editor.action.sourceAction', createConversionFunction(...[])];
+    mappings['editor.action.organizeImports'] = ['monaco.editor.action.organizeImports', createConversionFunction(...[])];
+    mappings['editor.action.autoFix'] = ['monaco.editor.action.autoFix', createConversionFunction(...[])];
+    mappings['editor.action.fixAll'] = ['monaco.editor.action.fixAll', createConversionFunction(...[])];
+    mappings['editor.action.rename'] = ['monaco.editor.action.rename', createConversionFunction(...[])];
+    mappings['editor.action.insertCursorAbove'] = ['monaco.editor.action.insertCursorAbove', createConversionFunction(...[])];
+    mappings['editor.action.insertCursorBelow'] = ['monaco.editor.action.insertCursorBelow', createConversionFunction(...[])];
+    mappings['editor.action.insertCursorAtEndOfEachLineSelected'] = ['monaco.editor.action.insertCursorAtEndOfEachLineSelected', createConversionFunction(...[])];
+    mappings['editor.action.addSelectionToNextFindMatch'] = ['monaco.editor.action.addSelectionToNextFindMatch', createConversionFunction(...[])];
+    mappings['editor.action.addSelectionToPreviousFindMatch'] = ['monaco.editor.action.addSelectionToPreviousFindMatch', createConversionFunction(...[])];
+    mappings['editor.action.moveSelectionToNextFindMatch'] = ['monaco.editor.action.moveSelectionToNextFindMatch', createConversionFunction(...[])];
+    mappings['editor.action.moveSelectionToPreviousFindMatch'] = ['monaco.editor.action.moveSelectionToPreviousFindMatch', createConversionFunction(...[])];
+    mappings['editor.action.selectHighlights'] = ['monaco.editor.action.selectHighlights', createConversionFunction(...[])];
+    mappings['editor.action.changeAll'] = ['monaco.editor.action.changeAll', createConversionFunction(...[])];
```

Comment:
Does the compiler complain about "createConversionFunction()", also, I think we can use a constant IDENTITY= createConversionFunction()